### ISSUE TITLE
Dependency on `toml` prevents using `version-sync` with MSRV older than that declared by `toml` crate

### ISSUE THREAD

**Issue by lopopolo** [mgeisler/version-sync#140]
See:

- https://github.com/artichoke/intaglio/pull/246

The `toml` crate has a bot configured on its repository to automatically update its MSRV field in `Cargo.toml`. Changes to this field, which affect the ability of cargo to resolve crate versions, are published in patch releases.

This transitive dependency means I cannot use `version-sync` in any codebases that have different/older MSRV policies than the `toml` crate.

`basic-toml` is an alternative crate which is a fork of the `toml` v0.5.x branch. However, it does not expose a "parse to `Value`" routine. One must go through `serde` to use it.

**Comment by lopopolo** [comment]
apologies, it looks like I was mistaken. Maintaining my dependency constraint on `version-sync` to be v0.9.3 as the lower bound allows `cargo` to do the right thing with Rust 1.58.0.

**Comment by mgeisler** [comment]
Hi @lopopolo! It sounds like we've both tried to maintain a MSRV for our projects... I tried to do this when Rust 2021 came out, but I gave up later: https://github.com/mgeisler/textwrap/pull/180.

In general, it seems to be a losing battle... transitive dependencies keep bumping their MSRV in patch releases. This breaks CI again and again, even when nothing changes in your own code. I'm now testing with stable and nightly for most of my crates.
